webpack.config.js

```diff
@@ -66,6 +66,13 @@ alias.config = path.resolve(
   "config.yml",
 );
 
+const mapProvider = process.env.MAP_PROVIDER || "mapbox-gl";
+alias.map = path.resolve(
+  __dirname,
+  "src/base/static/map-providers",
+  mapProvider + ".js",
+);
```

Comment:
> Will choosing the Map Provider be driven by the flavor's config, or will it only be available to custom-code flavors?

Ideally both. I think a Mapseed feature where users spin up new maps against a mapping provider of their choice would be cool.

> Also, what's the benefit of setting the map provider at build time instead of run time? 

I was hoping to avoid the need to bundle more than one mapping provider. But maybe that's not a huge concern, especially if we eventually split our vendor bundle from our application bundle.

Yeah, I think the current aliased build step could be replaced with a config flag and the application could sort out the map provider at run time. And I suppose it would need to be if it's going to be a platform feature. The only caveat is that eventually we'd be bundling unused map libraries.

But I think it makes sense-- I'll remove the webpack alias and replace it with a config flag that sets the mapping provider.

Reply:
Perhaps we can use the Mapbox provider as a default to avoid cluttering the config